packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -57,17 +63,12 @@ const double _kSegmentMinPadding = 9.25;
 // Both the mechanism and the value are speculated.
 const double _kTouchYDistanceThreshold = 50.0 * 50.0;
 
-// The corner radius of the segmented control.
-//
-// Inspected from iOS 13.2 simulator.
-const double _kCornerRadius = 8;
-
 // The minimum opacity of an unselected segment, when the user presses on the
 // segment and it starts to fadeout.
-//
-// Inspected from iOS 13.2 simulator.
```

Comment:
consider documenting the source of this const: https://github.com/flutter/flutter/issues/21302